cms/templates/settings.html

```diff
@@ -386,10 +386,43 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                     <span class="tip tip-stacked">${_("Enter your YouTube video's ID (along with any restriction parameters)")}</span>
                   </div>
                 </li>
-                % endif
+                  % endif
               </ol>
             </section>
 
+          % if enable_extended_course_details:
+            <hr class="divide" />
+            <section class="group-settings course-learning-info">
+              <header>
+                <h2 class="title-2">${_("Learning Outcomes")}</h2>
+                <span class="tip">${_("Add the learning outcomes for this course")}</span>
+              </header>
+              <ol class="list-input enum">
+                <li class="course-settings-learning-fields"></li>
+                <li>
+                  <a href="javascript:void(0)" class="action action-primary button new-button add-new-data add-course-learning-info">
```

Comment:
Instead of using anchor links, use buttons. Getting anchor links to properly handle accessibility (when behaving as a button) is very difficult, and you get it all for free with a button.